dotnet/Microsoft.MacCatalyst.Templates/maccatalyst/Main.cs

```diff
@@ -0,0 +1,15 @@
+﻿using UIKit;
+
+namespace MacCatalystApp1
+{
+    public class Application
+    {
+        // This is the main entry point of the application.
+        static void Main(string[] args)
```

Comment:
Do template code need to use a specific coding style? Or will they be reformatted according to the user's settings when they're instantiated?

Reply:
There is a `dotnet format` tool: https://www.hanselman.com/blog/editorconfig-code-formatting-from-the-command-line-with-net-cores-dotnet-format-global-tool

I tested this, I dropped the `.editorconfig` from [xamarin-android](https://github.com/xamarin/xamarin-android/blob/master/.editorconfig), then did:

```
% dotnet new ios
% dotnet format
```

When I saw other templates, they all use VS Windows coding style -- but I guess it could really be _any_ style? I think `dotnet format` would fix up anything.

Reply:
That tools looks pretty amazing!

Its mere existence makes me vote towards having the templates use our own coding style (in which case the coding style is wrong in some of the new files - I guess you could run `dotnet format` locally in the template directory and that'll fix it 😄 )